### PR TITLE
misc updates

### PR DESCRIPTION
**Pull request by wteouchicago**
- DOC-344 replaced old analysis center images in Sequence Reads and PP
- DOC-397 removes MAF aggregation refs
- added blank lines to get bullets to render correctly
- DOC-309 years-days conversion info in cDAVE
- DOC-404 adds info about behaviour of header checkbox

### CODE REVIEW

File: Data_Portal_UG.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ pages:
   - ProteinPaint: 'Data_Portal/Users_Guide/proteinpaint_lollipop.md'
   - Sequence Reads: 'Data_Portal/Users_Guide/proteinpaint_bam.md'
   - OncoMatrix: 'Data_Portal/Users_Guide/oncomatrix.md'
-  - Cohort MAF: 'Data_Portal/Users_Guide/cohortMAF.md'
+  # - Cohort MAF: 'Data_Portal/Users_Guide/cohortMAF.md'
```
Comment (wteouchicago) [inline Data_Portal_UG.yml:24]: @wwysoc2 please check that this is right/ok.

File: mkdocs.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ pages:
   - ProteinPaint: 'Data_Portal/Users_Guide/proteinpaint_lollipop.md'
   - Sequence Reads: 'Data_Portal/Users_Guide/proteinpaint_bam.md'
   - OncoMatrix: 'Data_Portal/Users_Guide/oncomatrix.md'
-  - Cohort MAF: 'Data_Portal/Users_Guide/cohortMAF.md'
+  # - Cohort MAF: 'Data_Portal/Users_Guide/cohortMAF.md'
```
Comment (wteouchicago) [inline mkdocs.yml:55]: @wwysoc2 please check that this is right/ok.

Comment (wwysoc2) [reply mkdocs.yml:55]: No idea if commenting out the line will work, let's merge it and see how it builds.

**Review by clsackett — APPROVED** [review]
LGTM!

**Review by wwysoc2 — APPROVED** [review]
Looks good. I'm going to merge and try to deploy.  I'll delete the mkdocs/PDF lines if something goes awry.

**Pull request merged**